### PR TITLE
Dashboard/Twitter cards previews not using CDN

### LINKED ISSUE #5926: Dashboard/Twitter cards previews not using CDN
I found some places using previews with named maps fixed URLs that are not using a CDN to request the images:

 1. `https://{user}.cartodb.com/dashboard` previews are using fixed URLs but not CDN ones.
 1. `https://{user}.cartodb.com/viz/{viz_id}/public_map` twitter card `meta name="twitter:image"` URL is using fixed URL but not CDN one.

Related to https://github.com/CartoDB/cartodb/issues/5637

### PR DESCRIPTION
**Pull request by Kartones**
Relates to #5926 (Solves scenario 1), as 2) is config-related )

### CODE REVIEW

File: lib/static_maps_url_helper.rb [hunk 1]
```diff
@@ -4,12 +4,14 @@ class StaticMapsURLHelper
 
     def url_for_static_map(request, visualization, map_width, map_height)
       username = CartoDB.extract_subdomain(request)
-      request_protocol = request.protocol.sub('://','')
-      static_maps_base_url(username, request_protocol) + static_maps_image_url_fragment(visualization.id, map_width, map_height)
+      request_protocol = request.protocol.sub('://', '')
```
Comment (Kartones) [inline lib/static_maps_url_helper.rb]: This lines are just formatting for Hound, can be ignored

File: config/app_config.yml.sample [hunk 2]
```diff
@@ -10,7 +10,7 @@ defaults: &defaults
   subdomainless_urls: false
   http_port:           3000 # nil|integer. HTTP port to use when building urls. Leave empty to use default (80)
   https_port:          # nil|integer. HTTPS port to use when building urls. Leave empty to use default (443)
-  maps_api_cdn_template: # nil|string. E.g. '{protocol}://{zone}.cartocdn/{user}/api/v1/map/static/named/{tpl_id}/{width}/{height}.png'
+  maps_api_cdn_template: # nil|string. E.g. '{protocol}://{zone}.cartocdn/{user}' (`/api/v1/...` fragment added via code)
```
Comment (Kartones) [inline config/app_config.yml.sample]: outdated and wrong doc...

**Comment by Kartones** [comment]
@xavijam CR please (as you'll see really is 3 lines of code regarding frontend config)

**Comment by xavijam** [comment]
Great :+1:, what about the other issue? Could we open a new one in platform for system engineers?

**Comment by Kartones** [comment]
I've assigned the ticket to Luis so he or somebody from his team works on it, I'd do the config change but I'm missing some internals details.

**Comment by xavijam** [comment]
ping @luisbosque

**Pull request merged**